data/brands/amenity/pub.json

```
@@ -57,6 +57,16 @@
         "brand:wikidata": "Q123913080"
       }
     },
+    {
+      "displayName": "Craft Union",
+      "locationSet": {"include": ["gb"]},
```

They only have pubs in England and Wales

```suggestion
      "locationSet": {"include": ["gb-eng", "gb-wls"]},
```

---

For now